### DISCUSSION TITLE
x86_64 Plans?

### DISCUSSION THREAD

**Discussion by lonnietc** [bao-project/bao-hypervisor#127] [General]
Hello,

I am researching separation kernels as well as other Type-1 hypervisors and came across Bao.

Was wondering if there were any plans to support the x86_64 platform or how hard it might be to add that in?

Thanks and have a great day

**Comment by sandro2pinto** [comment]
Hello @lonnietc,

There are no concrete plans in the open-source Bao roadmap to support x86_64.

To be honest, there are just two paths I see this happening (in the near future):
1. An interested 3rd party requesting that via a commercial agreement with OSYX (https://osyx.tech/);
2. An open-source contribution from a 3rd party, which we would happily support.

Thanks
Sandro

**Comment by josecm** [comment]
I'm moving this to the discussion. @lonnietc please also check my previous answer to a similar question in #88.
